docs/v1.0/metrics/domains.md

```diff
@@ -482,7 +490,17 @@ A comma-separated list of database or table names to exclude (ignored if `includ
 Default: `yes`<br>
 If the source table should be truncated to reset data after each retrieval.
 
+* `truncate-timeout`<br>
+Default: 2s<br>
```

Comment:
Let's be more conservative: 500ms. Truncate should be nearly instantaneous. Even 500ms is really long for this query. Other issue is that the whole level will wait for this, and 2s is a long wait (in this context).

Reply:
Maybe even 250ms. I'd measure how long it normally takes then do 2x or 3x that, to use something quasi-objective.

Reply:
60 - 80ms seems common. 250 seems like a reasonable default. We just need it to too many instances of the query stacking up, so as long as it is less than the sampling frequency it should be fine.